chia/cmds/farm_funcs.py

```diff
@@ -12,24 +12,26 @@
 from chia.util.default_root import DEFAULT_ROOT_PATH
 from chia.util.ints import uint16
 from chia.util.misc import format_minutes
+from chia.server.outbound_message import NodeType
 
 SECONDS_PER_BLOCK = (24 * 3600) / 4608
 
 
-async def get_plots(harvester_rpc_port: int) -> Optional[Dict[str, Any]]:
+async def get_plots(harvester_rpc_port: int, harvester_rpc_hostname: str) -> Optional[Dict[str, Any]]:
     plots = None
     try:
         config = load_config(DEFAULT_ROOT_PATH, "config.yaml")
-        self_hostname = config["self_hostname"]
+        if harvester_rpc_hostname is None:
+            harvester_rpc_hostname = config["self_hostname"]
```

Comment:
Could we add a check here to translate `localhost` into `0.0.0.0` so we don't need to modify our config to make this work?